### ISSUE TITLE
Full-Width tabs disabled

### ISSUE THREAD

**Issue by peteringram0** [arcticicestudio/nord-atom-ui#57]
When the full-width tabs are disabled the close button is on-top of the file type icon.

<img width="109" alt="screen shot 2017-03-02 at 08 33 28" src="https://cloud.githubusercontent.com/assets/6588942/23499166/f6855a24-ff22-11e6-8f93-42d8b34110ba.png">

**Comment by arcticicestudio** [comment]
Thanks for your report 👍 
I tried to reproduce this but it seems like there are differences in the setup.
I am running Atom 1.14.3 with the [`file-icons`](https://atom.io/packages/file-icons) package version `2.0.17` and my icon for the [`settings-view`](https://atom.io/packages/settings-view) is different as you can see on the screenshot.

![ghi-57-scrot-home-setup](https://cloud.githubusercontent.com/assets/7836623/23528004/75d16eba-ff98-11e6-856e-9fc66142bca8.png)


Can you please post some setup information like
  - the version of Atom
  - a list of all your installed packages including the version to see if there are conflicts
  - the OS you're running
    - if it is *not* Windows please add information about your distro

so I can try to reproduce it?

**Comment by peteringram0** [comment]
I had the seti-icons package enabled which was causing part of the issue. With it disabled it still looks like there is an issue:

<img width="143" alt="screen shot 2017-03-03 at 21 34 33" src="https://cloud.githubusercontent.com/assets/6588942/23569817/3d8f795e-0059-11e7-86e9-f06823c5f557.png">

Atom version: 1.14.4
Packages: [stylus, atom-beautify, atom-bootstrap3, atom-jade, atom-typescript, dotblocker, editorconfig, emmet, file-icons (2.0.17), fold-functions, pigments]
OSX 10.12.3

````javascript
"*":
  "atom-beautify":
    general:
      _analyticsUserId: "59b34174-1565-4602-a67e-9f89ddecde22"
  core:
    disabledPackages: [
      "goto-definition"
      "language-elm"
      "language-elmx"
      "linter"
    ]
    telemetryConsent: "no"
    themes: [
      "nord-atom-ui"
      "one-dark-syntax"
    ]
  editor:
    fontSize: 12
    preferredLineLength: 100
    showIndentGuide: true
    tabLength: 4
  "exception-reporting":
    userId: "e94c6348-7cc9-b2b6-df64-88867be0175f"
  minimap:
    absoluteMode: true
  "nord-atom-ui":
    compactView: true
    fileIcons: false
    tabSizing: false
  "one-light-ui":
    layoutMode: "Compact"
  welcome:
    showOnStartup: false
````

**Comment by arcticicestudio** [comment]
Thanks, I've tested this on macOS. I forgot that I've implemented [this selector](https://github.com/arcticicestudio/nord-atom-ui/blob/master/styles/tabs.less#L140#L144) to move the close icon on tabs to the left. This can be fixed by setting `padding-left: 10px` for the icon.
Following these conversations in [atom/tabs #378](https://github.com/atom/tabs/issues/378) and [atom/one-dark-ui #179](https://github.com/atom/one-dark-ui/issues/179), this should be better moved into a optional theme setting to not mess with people's muscle memory. Maybe this will be implemented into the core 📦  [atom/tabs](https://github.com/atom/tabs) sometime.

**Comment by arcticicestudio** [comment]
I played around with the tab sizing rules and I'll just remove the platform-specific selector for now because a theme setting would cause more bugs.
It will be reimplemented after a whole refactoring of the UI sizing code. (#50)

**Comment by peteringram0** [comment]
When will this be merged in plz? Thanks 👍

**Comment by arcticicestudio** [comment]
I'll do it this evening when I'm back home 😄

**Comment by arcticicestudio** [comment]
<p align="center"><strong>Before with <code>file-icons</code> package</strong><br><img src="https://cloud.githubusercontent.com/assets/7836623/23630275/d2df037c-02ba-11e7-9948-32f0dae70c6d.png"/><br><strong>Before with <code>seti-icons</code> package</strong><br><img src="https://cloud.githubusercontent.com/assets/7836623/23630292/e2e3dfae-02ba-11e7-9f5c-2a261c93c499.png"/></p>

<p align="center"><strong>After with <code>file-icons</code> package</strong><br><img src="https://cloud.githubusercontent.com/assets/7836623/23630324/016f325c-02bb-11e7-8ad0-e295446f6eff.png"/><br><strong>After with <code>seti-icons</code> package</strong><br><img src="https://cloud.githubusercontent.com/assets/7836623/23630345/183d8a88-02bb-11e7-8214-516bc35fcb76.png"/></p>

**Comment by arcticicestudio** [comment]
🚢 Shipped in `apm` package release version 🏷 [0.10.2](https://github.com/arcticicestudio/nord-atom-ui/releases/tag/v0.10.2)

/cc @peteringram0

**Comment by peteringram0** [comment]
Thanks, works like a charm :)
